sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/utils.scala

```diff
@@ -81,20 +81,38 @@ object Utils {
       groupingExpressions: Seq[NamedExpression],
       aggregateExpressions: Seq[AggregateExpression],
       resultExpressions: Seq[NamedExpression],
+      partialAggregation: Boolean,
```

Comment:
Partial aggregation IMO implies that we add a partial aggregation step. What do you think?